JumpscaleLibs/tools/tf_gateway/TFGateway.py

```diff
@@ -4,6 +4,7 @@
 
 
 JSBASE = j.baseclasses.object
+DNS_PREFIX = j.core.myenv.config.get("DNS_PREFIX", "")
```

Comment:
why not use TFGatway constructor or even do `dns_prefix_set` in the tool itself ?